lib/get-config.js

```diff
@@ -27,16 +27,35 @@ const ALLOWED_ERROR_CODES = new Set([
 async function requirePlugin(workingDir, pluginName, fromConfigPath) {
   let basedir = fromConfigPath === undefined ? workingDir : path.dirname(fromConfigPath);
 
+  let plugin;
+
+  try {
+    let importedModule = await import(pluginName);
```

Comment:
Without _trying_ this first, we can't actually use plugins from `node_modules` :see_no_evil: 

For example, prior to this change, we'd get this error:
```
<repo>/node_modules/resolve/lib/sync.js:102
    var err = new Error("Cannot find module '" + x + "' from '" + parent + "'");
              ^

Error: Cannot find module 'ember-template-lint-plugin-tailwindcss' from '<repo>/frontend'
    at Function.resolveSync [as sync] (<repo>/node_modules/resolve/lib/sync.js:102:15)
    at requirePlugin (file://<repo>/node_modules/ember-template-lint/lib/get-config.js:31:28)
    at processPlugins (file://<repo>/node_modules/ember-template-lint/lib/get-config.js:150:22)
    at getProjectConfig (file://<repo>/node_modules/ember-template-lint/lib/get-config.js:396:28)
    at async Linter.loadConfig (file://<repo>/node_modules/ember-template-lint/lib/linter.js:48:19)
    at async Linter.verify (file://<repo>/node_modules/ember-template-lint/lib/linter.js:304:7)
    at async run (file://<repo>/node_modules/ember-template-lint/bin/ember-template-lint.js:211:21) {
  code: 'MODULE_NOT_FOUND'
}
error Command failed with exit code 1.    }

```

Reply:
Can you explain more about how the monorepo is setup?

Reply:
The issue is present in _any_ monorepo where the ember project is not at the root, where the root's node_modules contains ember-template-lint (so almost _every_ monorepo)

Reply:
Are you suggesting that your monorepo has `ember-template-lint` in the root (presumably as a devDep), and the plugins in subdirs?

Reply:
Just that Monorepo hoisting can place the package anywhere, we can't assume the location

Reply:
Sure, but typically the idea in a monorepo is to keep your tool devDeps in the root package.json. If you did this would this not work for you?

Reply:
OK sure. I suppose from my perspective that's part of the benefit of monorepos - minimizing tool dependencies so you have less maintenance. But of course that's not super relevant, though what I was getting at was trying to first determine if this was an issue that could be handled via a recommended setup vs. a code change.

Reply:
makes sense -- important question though -- why were we _not_ relying on the built-in node resolution algo and trying to find deps ourselves?

Reply:
> why were we not relying on the built-in node resolution algo (which happens for us via import/require) and trying to find deps ourselves?

Because the node resolution algorithm is actually wrong in this case 😸. Specifically, the node modules algorithm would suggest that the plugin **must** be resolvable from whatever location `ember-template-lint` itself is installed at. **Which is not correct!** The plugin will be installed somewhere relative to the `.template-lintrc.js` file **not** where ever** `ember-template-lint` happens to be installed. This is the point of the current `resolve.sync` code.